README.md

```diff
@@ -13,30 +11,40 @@ Devices connected to IoT Hub can de described with DTDL, in terms of:
 - Commands (Using the Hub Direct Methods feature)
 - Telemetry (processed through EventHub)
 
-Review the docs [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=release-preview-refresh-iot-pnp).
+Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114283)
 
-### Private environments, and preview SDKs
+### BugBash support and feedback
 
-As the time of writing the version of IoT Hub supporting PnP BugBash 2, 08 May2020 release is only deployed to a private hub instance:
+- The teams channel [PnP Public Preview BugBash](https://teams.microsoft.com/l/channel/19%3a0b9d0f166a3d41c69ce90fcca7631962%40thread.skype/PnP%2520Public%2520Preview?groupId=dcc1ac84-f476-4c96-8034-b2d77e54c8bf&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47). Will have team members answering questions during the BugBash.
+- If you have bugs or feature request, please use this [Bug Template](https://msazure.visualstudio.com/One/_workitems/create/Bug?templateId=a05dbe62-eb63-4028-9ce0-252c552872f3&ownerId=1e65a829-00c0-4dc9-8088-d41678a0d033). You can query existing bugs in this [PnP BugBash Query](https://msazure.visualstudio.com/One/_queries/query-edit/71cca2fe-63bd-4916-84f6-937f0dfaa698/?action=new)
+- For feedback related to the docs, you can use this [azure docs PR](https://github.com/MicrosoftDocs/azure-docs-pr/pull/114283)
 
-```js
-<Connection Strings for IoT Hub>
-```
+
+## Required environments, SDKs and tools
+
+### IoT Hub
+
+The required hub version is only  available in the canary environment, to create a new hub follow these steps:
+
+- Request access to the `IOTPNP_TEST_BY_MAIN` subscription, via the Teams channel
```

Comment:
@minune-msft can you review if these instructions are enough to create hubs in canary?